newrelic/resource_newrelic_nrql_alert_condition_test.go

```diff
@@ -84,7 +83,7 @@ func TestAccNewRelicNrqlAlertCondition_Basic(t *testing.T) {
 // TODO: func_ TestAccNewRelicNrqlAlertCondition_Multi(t *testing.T) {
```

Comment:
Fun fact: that comment is actually an artifact from me originally copying this: https://github.com/terraform-providers/terraform-provider-newrelic/blob/master/newrelic/resource_newrelic_nrql_alert_condition_test.go#L84

So that test isn't actually relevant for Infra as far as I can think. I will cut that out of here and resubmit.

Reply:
I am actually unsure what the intention of that test case was as it isn't related to my changes or Infra. It looks like it was also copy pasted from a different test file: https://github.com/terraform-providers/terraform-provider-newrelic/blob/master/newrelic/resource_newrelic_alert_condition_test.go#L111

Reply:
Fair enough (see the some other `TODOs` as well). Approving this and merging.